server/data/tokenStore/tokenStoreFactory.ts

```diff
@@ -0,0 +1,18 @@
+import config from '../../config'
+import { createRedisClient } from '../redisClient'
+import InMemoryTokenStore from './inMemoryTokenStore'
+import RedisTokenStore from './redisTokenStore'
+import TokenStore from './tokenStore'
+
+let inMemoryTokenStore: InMemoryTokenStore
+
+const tokenStoreFactory = (prefix: string): TokenStore => {
```

Comment:
Adding this so the in-memory vs Redis store logic doesn't need to be repeated wherever a store is created.